src/Shouldly.Tests/ShouldNotContain/StringContainsStringCaseIsInsensitiveScenario.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.ShouldNotContain
+{
+    public class StringContainsStringCaseIsInsensitiveScenario : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldThrowAWobbly()
+        {
+            "Shouldly is legendary".ShouldNotContain("legendary", Case.Insensitive);
```

Comment:
Hey,

Yeah you can create a new context type, like we do in https://github.com/shouldly/shouldly/blob/master/src/Shouldly/Internals/ShouldlyCoreExtensions.cs#L55 which holds the values properly